CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/RequestHandler/ReplicatorConfigurationRequestHandler.java

```diff
@@ -256,5 +285,41 @@ public void setReplicatorType(Args args){
         byte[] bytes = bos.toByteArray();
         return bytes;
     }
+}
+
+class ReplicatorBooleanFiltlerCallback implements ReplicationFilter {
+    @Override
+    public boolean filtered(Document document, EnumSet<DocumentFlag> flags) {
+        String key = "new_field_1";
```

Comment:
I'm using this constant value as an additional field in my update. This check is intended for those update only, hence it would never change to any other key.